test/e2e_node/resource_metrics_test.go

```diff
@@ -64,13 +63,13 @@ var _ = framework.KubeDescribe("ResourceMetricsAPI", func() {
 			ginkgo.By("Waiting 15 seconds for cAdvisor to collect 2 stats points")
 			time.Sleep(15 * time.Second)
 		})
-		ginkgo.It("should report resource usage through the v1alpha1 resouce metrics api", func() {
```

Comment:
Add another clause like this for v1alpha1? Same test, different endpoints?

Reply:
Yes.  #88568 was supposed to keep the contents of the endpoint the same IIRC, but just refactor and add a new endpoint.

Reply:
> Yes. #88568 was supposed to keep the contents of the endpoint the same IIRC, but just refactor and add a new endpoint.

Yes.